.github/workflows/android.yml

```diff
@@ -105,7 +105,7 @@ jobs:
         if: ${{ endswith(env.repo, '/cordova-paramedic') != true }}
         run: npm i -g github:apache/cordova-paramedic
 
-      - uses: reactivecircus/android-emulator-runner@5de26e4bd23bf523e8a4b7f077df8bfb8e52b50e
+      - uses: reactivecircus/android-emulator-runner@d94c3fbe4fe6a29e4a5ba47c12fb47677c73656b
```

Comment:
Any reason we use specific commits over released versions and their tags?

Reply:
This was an Apache requirement.  They allow listed only some scopings.

For example:

* `actions/<action-name>`
* `apache/<action-name>`

Maybe a few others like Microsoft etc...

But as for those that were not allowed, I was told to use the commit hash ID.

Why they implemented it this way I am unsure. I was told this I beleive in Apache's Infra Slack Channel.

Reply:
Thanks for the explanation!